### ISSUE TITLE
Add a 'move' and possibly 'duplicate' shortcut so I can quickly relocate my currently open file

### ISSUE THREAD

**Issue by ProLoser** [Trudko/advanced-new-file#28]
Instead of making a new file from nothing, it'd be nice if the currently open file could be relocated using the same exact prompt as you would with a new file.

**Comment by iamstarkov** [comment]
are you talking about renaming?

**Comment by ProLoser** [comment]
That would work too, yes. I suppose it'd be easy to select-all, copy, new file, paste

**Comment by ProLoser** [comment]
Yes quick-renaming would be awesome if it had the exact same UI.

**Comment by iamstarkov** [comment]
can you describe cases and the flow of anf’s work in these cases

**Comment by ProLoser** [comment]
anf?

Uh all I'm thinking is that you open a file, while that file is the current editor you hit some shortcut like cmd+option+r which opens the 'new file' prompt, except the full path to the file is currently shown and the filename is selected. You can then begin typing in a new name or path/to/new/name.file

**Comment by iamstarkov** [comment]
I get your idea of renaming scenario, seems to be useful. thx

**Comment by Trudko** [comment]
I am not sure that I follow. So do you want to be able to open ANF and if you have some file open it's path would be in dialog and you could rename it from there or be able to write new path and the file which is right now opened would be moved there?

**Comment by ProLoser** [comment]
I didn't think this was that confusing.

In the file-tree, if you highlight a file or folder, and press "m" you will get a move dialogue. I pretty much want that, except while the editor is in focus, and it would use the currently open file and the autocomplete menu. Honestly it would also be nice if you retroactively added the autocomplete menu to the existing dialogues that are initiated from the file tree.
